arangod/ClusterEngine/ClusterIndex.cpp

```diff
@@ -150,7 +150,7 @@ void ClusterIndex::toVelocyPack(
       std::string const url =
           prefix + shard.first + "/" + std::to_string(_iid.id());
       futures.emplace_back(
-          network::sendRequestRetry(pool, "shard:" + shard.first,
+          network::sendRequestRetry(pool, "shard:" + std::string(shard.first),
```

Comment:
Given that there is an operator+ defined for ShardID, do we need the explicit conversion to an std::string here?

Reply:
Right, done